### PR TITLE
Add audits folder

### PR DESCRIPTION
**Pull request by jubeira**
# Description

Cantina final audit report pending.

## Type of change

- [ ] Bug fix 
- [ ] New feature 
- [ ] Breaking change 
- [ ] Dependency changes
- [ ] Code refactor / cleanup
- [ ] Optimization: [ ] gas / [ ] bytecode
- [x] Documentation or wording changes
- [ ] Other

## Checklist:

- [x] The diff is legible and has no extraneous changes
- N/A Complex code has been commented, including external interfaces
- N/A Tests have 100% code coverage
- [x] The base branch is either `main`, or there's a description of how to merge

## Issue Resolution

N/A

### CODE REVIEW

File: audits/README.md [hunk 1]
```diff
@@ -0,0 +1,14 @@
...
+This directory the reports of audits performed on Balancer smart contracts by different security firms.
+The audits have been conducted before the [Cantina contest](https://cantina.xyz/competitions/949ad7c5-ea14-427d-b10a-54e33cef921b), and all the relevant findings addressed.
+
+| :warning: | Audits are not a guarantee of correctness. Some of the contracts were modified after they were audited.      |
+| --------- | :----------------------------------------------------------------------------------------------------------- |
+
+| Scope                             | Firm          | Report                                         |
+| --------------------------------- | ------------- | ---------------------------------------------- |
+| Vault, Weighted Pool, Stable Pool | Certora       | [`2024-09`](./certora/2024-09.pdf)             |
+| Vault, Weighted Pool, Stable Pool | Trail Of Bits | [`2024-10`](./trail-of-bits/2024-10.pdf)       |
```
Comment (EndymionJkb) [inline audits/README.md:14]: I don't have a strong preference, but we did use the full date in the corresponding v2 file. Would be 2024-09-04 for Certora, 2024-10-08 for ToB.

Comment (EndymionJkb) [reply audits/README.md:14]: We're waiting on Spearbit as noted. What about the 4th one?

**Review by EndymionJkb — APPROVED** [review]
We'll have to figure out what we're doing with the deployments repo, and add a link to that (maybe separate for v2 and v3)?

Maybe we won't need the Cantina contest link later (pending results I guess)?

Anyway we can add the directory.

**Review by joaobrunoah — APPROVED** [review]
LGTM!

File: audits/README.md (hunk 1 above)
Comment (jubeira) [reply audits/README.md]: Could be, yes. Merging this one for now, but it could be interesting to push that one too.

**Pull request merged**